plugins/jobtype/src/azkaban/jobtype/HadoopJavaJob.java

```diff
@@ -121,8 +121,27 @@ protected String getJVMArguments() {
     classPath.add(HadoopConfigurationInjector.getPath(getJobProps(),
         getWorkingDirectory()));
 
-    List<String> typeClassPath =
-        getSysProps().getStringList("jobtype.classpath", null, ",");
+    // merging classpaths from plugin.properties
+    mergeClassPaths(classPath,
+      getJobProps().getStringList("jobtype.classpath", null, ","));
+    // merging classpaths from private.properties
+    mergeClassPaths(classPath,
+      getSysProps().getStringList("jobtype.classpath", null, ","));
+
+    List<String> typeGlobalClassPath =
+        getSysProps().getStringList("jobtype.global.classpath", null, ",");
+    if (typeGlobalClassPath != null) {
```

Comment:
As mentioned before, this is an existing part of code and we are trying to minimize changes (refer L141-L151).